pkgs/by-name/yu/yunfaavatar/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, makeWrapper
+, pciutils
+, x11Support ? true
+, ueberzug
+, fetchpatch
+, pkgs
```

Comment:
Please don't use `pkgs` here. Individual inputs should be listed out (bash and imagemagick) to allow overriding